clang/include/clang/Basic/AArch64SVEACLETypes.def

```diff
@@ -57,6 +57,11 @@
 //  - IsBF true for vector of brain float elements.
 //===----------------------------------------------------------------------===//
 
+#ifndef SVE_SCALAR_TYPE
+#define SVE_SCALAR_TYPE(Name, MangledName, Id, SingletonId, Bits) \
```

Comment:
Why not AARCH64, as this is not an SVE type. At least that was the suggestion in the past

Reply:
No reason. Can be AARCH64.